test/Yoti.Auth.Tests/ApplicationProfileTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Yoti.Auth.Tests.TestTools;
+using static Yoti.Auth.YotiAttributeValue;
+
+namespace Yoti.Auth.Tests
+{
+    [TestClass]
+    public class ApplicationProfileTests
+    {
+        private string _value = "value";
+
+        [TestMethod]
+        public void ApplicationProfile_NameAttribute()
+        {
+            ApplicationProfile applicationProfile = TestTools.Profile.CreateApplicationProfileWithSingleAttribute(
```

Comment:
Don't we have a mocking framework?

Reply:
I have created https://lampkicking.atlassian.net/browse/SDK-656 to introduce this